packages/stores/src/queries-external/token-data/index.ts

```diff
@@ -2,6 +2,7 @@ import { KVStore } from "@keplr-wallet/common";
 import { HasMapStore } from "@keplr-wallet/stores";
 import { Dec, RatePretty } from "@keplr-wallet/unit";
 import { computed, makeObservable } from "mobx";
+import { IPriceStore } from "src/price";
```

Comment:
careful with src imports, I've found them to make webpack unstable

Reply:
Nice catch!